### ISSUE TITLE
[BUG] Currency parsing error

### ISSUE THREAD

**Issue by ftp27** [alexanderepstein/Bash-Snippets#7]
**Issue Label:**
* [x] bug
* [ ] feature
* [ ] enhancement

**Description of bug or feature enhancement:**

```
currency
What is the base currency: EUR
What currency to exchange to: RUB
What is the amount being exchanged: 1600

=========================
| EUR to RUB
| Rate: 6.553
| EUR: 1600
| RUB: 10484.800
=========================
```
But from:
http://api.fixer.io/latest?base=EUR&symbols=RUB
```json
{"base":"EUR","date":"2017-06-27","rates":{"RUB":66.553}}
```

**OS and OS version:**
* [x] Mac
* [ ] Linux 32 Bit
* [ ] Linux 64 Bit
* [ ] Windows 32 Bit
* [ ] Windows 64 Bit

OS Version:

**Comment by alexanderepstein** [comment]
I see the issue in my code, will fix later today

**Comment by alexanderepstein** [comment]
This should be fixed with 4ddcc644e4e2f8d3f91dd501ef35cb5750f0d863

**Comment by alexanderepstein** [comment]
Will release update later tonight
